ioctl_list/src/main.cc

```diff
@@ -15,6 +15,9 @@
 
 #ifdef HAVE_NET_IF_H
 # include <net/if.h>
+#ifdef __DragonFly__
```

Comment:
@mneumann That's what autoconf takes care of. The autoconf philosophy is to test for features instead of platforms:

> they do not normally even need an argument specifying the system type. Instead, they individually test for the presence of each feature that the software package they are for might need

https://www.gnu.org/software/autoconf/manual/autoconf-2.62/html_node/Introduction.html#Introduction

If you're interested, I made some commits on top of your branch that do this with `AC_CHECK_HEADERS`: https://github.com/dcuddeback/ioctl-rs/compare/feature/dragonfly?expand=1#diff-ecc1a87332cffa717e3a14c626ae27cbL28